[33기 김형겸] ADD: LoginUI 레이아웃, styled-component 구현

Kyeom1997 opened the pull request:
Modal 창 레이아웃 구현, 각 Social Login Button 레이아웃 및 styled-components 구현
로그인 Component Left Side Carousel React slick을 통한 구현

## :: 최근 작업 주제 (하나 이상의 주제를 선택해주세요.)
- [x] 기능 추가
- [ ] 리뷰 반영
- [ ] 리팩토링
- [ ] 버그 수정
- [ ] 컨벤션 수정

<br />

## :: 구현 목표 (해당 브랜치에서 구현하고자 하는 하나의 목표를 설정합니다.)
- Login Component 레이아웃 및 styled-components 구현

<br />

## :: 구현 사항 설명 (작업한 내용을 상세하게 기록합니다.)
1. Login Component(Modal) 레이아웃 및 styled-components 구현
2. Login Component Left-Side Carousel (React-slick library) 통해 구현


<br />

## :: 성장 포인트 (해당 기능을 구현하며 고민했던 사항이나 새로 알게된 부분, 어려웠던 점 등을 작성합니다.)
- React-slick 사용법을 보다 자세하게 알게 되었습니다

<br />

## :: 기타 질문 및 특이 사항

hsuj86 commented on src/pages/Login/Login.js:
```diff
@@ -0,0 +1,229 @@
+            </LoginSlider>
+          </LoginCarousel>
+          <LoginStart>
+            <AiOutlineClose className="quit" size={30} />
```
- className 수정해 주세요.

Kyeom1997 replied:
수정 완료했습니다!

hsuj86 commented on src/pages/Login/Login.js:
```diff
@@ -0,0 +1,229 @@
+            <StartBox>
+              <strong>브랜치타임 시작하기</strong>
+            </StartBox>
+            <KakaoBox>
+              <span>
+                <RiKakaoTalkFill size={25} />
+              </span>
+              <SocialSpan>카카오 계정으로 시작하기</SocialSpan>
+            </KakaoBox>
+            <GithubBox>
+              <span>
+                <AiFillGithub size={25} />
+              </span>
+              <SocialSpan>깃허브 계정으로 시작하기</SocialSpan>
+            </GithubBox>
+            <FacebookBox>
+              <span>
+                <AiFillFacebook size={25} />
+              </span>
+              <SocialSpan>페이스북 계정으로 시작하기</SocialSpan>
+            </FacebookBox>
+            <NaverBox>
+              <span>
+                <SiNaver size={25} />
+              </span>
+              <SocialSpan>네이버 계정으로 시작하기</SocialSpan>
+            </NaverBox>
+            <GoogleBox>
+              <span>
+                <FcGoogle size={25} />
+              </span>
+              <SocialSpan>구글 계정으로 시작하기</SocialSpan>
+            </GoogleBox>
```
- 이것도 반복되는 부분이라 map()으로 사용할 수 있겠네요.

Kyeom1997 replied:
수정 완료했습니다!

hsuj86 commented on src/pages/Login/Login.js:
```diff
@@ -0,0 +1,229 @@
+              <div>
+                <LoginImg src="images/bread.png" alt="login" />
+                <strong>개발자로 데뷔하세요</strong>
+                <p>안녕하세요 브런치 싫어!</p>
```
- 반복적으로 사용되는데 map을 사용할 수 있지 않을까요?

hsuj86 reviewed (commented):
- 형겸님 고생하셨습니다! 리뷰 확인해 주세요.

Kyeom1997 committed:
ADD: Nav - Login Modal 기능 구현

1. Navbar에서 시작하기 버튼 클릭시에 setLoginModal 함수를 통해 loginModal false -> true로 state update
2. loginModal true 일시에 Login Component 표시
3. Login Modal Component에서 quit 버튼 클릭시에 다시 loginModal state false 값으로 변환 후에 Modal 창 사라짐

Modify: Conflict Resolve

hsuj86 commented:
- LGTM 👍🏻👍🏻👍🏻
